lib/database/ticket.php

```diff
@@ -12,11 +14,35 @@
 
 function isAllowedToSubmitTickets($user): bool
 {
-    return isValidUsername($user)
-        && getUserActivityRange($user, $firstLogin, $lastLogin)
+    if (!isValidUsername($user)) {
+        return false;
+    }
+
+    $key = $user . ':canTicket';
```

Comment:
```suggestion
    $key = 'user:' . $user . ':canTicket';
```

Prefix all keys the same way: https://github.com/RetroAchievements/RAWeb/blob/c6162d62743d51a849c3794e869b0d8d7ba29b01/lib/render/user.php#L76.
Allows to delete all keys by pattern if needed:
```php
Redis::del(Redis::keys('user:' . $user . ':*'));
```